lib/fastlane/plugin/wpmreleasetoolkit/actions/common/check_translation_progress.rb

```diff
@@ -0,0 +1,135 @@
+module Fastlane
+  module Actions
+    class CheckTranslationProgressAction < Action
+      def self.run(params)
+        require_relative '../../helper/glotpress_helper.rb'
+
+        UI.message('Check translations status...')
+
+        under_threshold_langs = check_translations(
+                                  glotpress_url: params[:glotpress_url], 
+                                  language_codes: params[:language_codes],
+                                  abort_on_violations: params[:abort_on_violations],
+                                  threshold: params[:min_acceptable_translation_percentage])
+
+        check_results(
+          under_threshold_langs: under_threshold_langs, 
+          threshold: params[:min_acceptable_translation_percentage], 
+          skip_confirm: params[:skip_confirm]
+        ) unless under_threshold_langs.length == 0
+
+        UI.message('Done')
+      end
+
+      # Check the status of the translations and returns the list of violations
+      # 
+      # @param [String] URL to the GlotPress project.
+      # @param [String] The list of codes (in GlotPress format) of the languages to check.   
+      # @param [Bool] Whether to abort on the first found violation or not.
+      # @param [Integer] The minimum acceptable percentage of translated strings.
+      #
+      # @return [Array] The list of violations (array of hashes of "language code" and "current percentage")
+      #
+      def self.check_translations(glotpress_url:, language_codes:, abort_on_violations:, threshold:)
+        under_threshold_langs = []
+
+        data = Fastlane::Helper::GlotPressHelper.get_translation_status_data(glotpress_url: glotpress_url) rescue nil
+        UI.abort_with_message!("Can't retrieve data from #{glotpress_url}") if data.nil? || data.length == 0
+
+        language_codes.each do | language_code |
+          progress = Fastlane::Helper::GlotPressHelper.get_translation_status(
+                                      data: data,
+                                      language_code: language_code) rescue -1
+          
+          if (abort_on_violations)
+            UI.abort_with_message!("Can't get data for language #{language_code}") if (progress == -1) 
+            UI.abort_with_message!("#{language_code} is translated #{progress}% which is under the required #{threshold}%.") if (progress < threshold) 
```

Comment:
Nitpick. How do you think this reads?

```suggestion
            UI.abort_with_message!("#{language_code} is #{progress}% translated, which is under the required #{threshold}% threshold.") if (progress < threshold) 
```